src/java.base/share/classes/jdk/internal/io/JdkConsoleImpl.java

```diff
@@ -57,6 +57,39 @@ public Reader reader() {
         return reader;
     }
 
+    @Override
+    public JdkConsole println(Object obj) {
+        pw.println(obj);
+        // automatic flushing covers println
+        return this;
+    }
+
+    @Override
+    public JdkConsole print(Object obj) {
+        pw.print(obj);
+        pw.flush(); // automatic flushing does not cover print
+        return this;
+    }
+
+    @Override
+    public String readln(String prompt) {
+        String line = null;
+        synchronized (writeLock) {
+            synchronized(readLock) {
```

Comment:
```suggestion
            synchronized (readLock) {
```

Reply:
Sorry, but no. It would be inconsistent with the rest of the file, which for whatever reason (copy-paste?) does not use whitespace between `synchronized` and `(readLock)`.

Reply:
It looks strange to have space in one case, and don't have it in another. I would'nt call it consistent.